browser-sign-in/app/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 
 Properties properties = new Properties()
-properties.load(project.rootProject.file('testing.properties').newDataInputStream())
+properties.load(project.rootProject.file('local.properties').newDataInputStream())
```

Comment:
have to check if the prop file exist. Travis runs unit test only so can put a check if it exists.